src/config.ts

```diff
@@ -27,6 +27,11 @@ export class CacheConfig {
   /** The workspace configurations */
   public workspaces: Array<Workspace> = [];
 
+  /** The max timeout for the networking operations */
+  public timeout: null | number = null;
```

Comment:
Same here, I think a default value would be useful.

Typically, GitHub downloads the cache with > 100MB/s. Considering that the cache limit is 10GB per repository, downloading a cache shouldn't take much longer that 100 seconds. What about a default value of 2minutes?